### PR TITLE
deleted database package

### PR DESCRIPTION
**Pull request by ElliotB-96**
deleted code for code review

### CODE REVIEW

**Comment by AdamKane44** [comment]
Didn't even need this anyway

File: src/database/DatabaseConnector.java [hunk 1]
```diff
@@ -1,185 +0,0 @@
...
-            String host = s.next();
-
-            DatabaseConnector.initConnection(user, pass, host);
-            System.out.println("Log in successful");
-            return true;
-
-        }catch (SQLException ex){
-
-            System.out.println("log in failed");
-            return false;
```
Comment (pwright08) [inline src/database/DatabaseConnector.java]: what is a false?

Comment (ElliotB-96) [inline src/database/DatabaseConnector.java]: 42

Comment (andyrobtaylor) [inline src/database/DatabaseConnector.java]: what is love?

**Pull request closed without merging**